### ISSUE TITLE
python 3.11 CI test failure

### ISSUE THREAD

**Issue by kecnry** [astropy/specreduce#227]
`specreduce/tests/test_synth_data.py::test_make_2d_arc_pass_wcs` is now failing with:

```
astropy.units.core.UnitConversionError: Can only apply 'greater' function to dimensionless quantities when other argument is not a quantity (unless the latter is all zero/infinity/nan).
```

during a call to specutils

```
../../specreduce/utils/synth_data.py:290: in make_2d_arc_image
    linelist = load_pypeit_calibration_lines(linelists, wave_air=wave_air)
../../specreduce/calibration_data.py:288: in load_pypeit_calibration_lines
    linelist['wavelength'] = vac_to_air(linelist['wavelength'])
../../.tox/py311-test-alldeps-cov/lib/python3.11/site-packages/specutils/utils/wcs_utils.py:148: in vac_to_air
    refr = refraction_index(wavelength, method=method, co2=co2)
../../.tox/py311-test-alldeps-cov/lib/python3.11/site-packages/specutils/utils/wcs_utils.py:103: in refraction_index
    if method != 'Griesen2006' and wavelength.min() < 200 * u.nm:
```

**Comment by pllim** [comment]
Is this `linelist['wavelength']` not Quantity?

**Comment by hpparvi** [comment]
This is now fixed in PR #231.
